02.extensions/zooseo/01.basics/06.faq/docs.md

```diff
@@ -0,0 +1,7 @@
+---
+title: F.A.Q.
+taxonomy:
+    category: docs
+---
+
+* **Is ZOOseo compatible with the other Joomla SEO extensions?** <br /> Yes, ZOOSeo extension works with any Joomla SEO extensions (for example, SH404SEF). They are complementary and work together.
```

Comment:
Remove **the** from "... compatible with **the**" and fix the S type from **ZOOSeo**.

Also remove the SH404SEF reference.